packages/interactive-messages/src/adapter.ts

```diff
@@ -486,6 +543,52 @@ export class SlackMessageAdapter {
         }
       }
 
+      if (constraints.handlerType === StoredConstraintsType.ViewSubmission) {
```

Comment:
These checks seem correct, but this and the ViewClosed constraints are the same. Is there a reason we can't combine them to reduce code duplication?

Reply:
There's a slight difference which is the first conditional within each section. The first checks for `view_submission` and the second checks for `view_closed`. But now that you point it out there's likely a way to reduce the code duplication - thanks!